### ISSUE TITLE
use of == for string comparison can be dangerous

### ISSUE THREAD

**Issue by lfoppiano** [Marame/M2Internship_Entity_Resolution#7]
We've discussed this already :-) 

```
    public boolean equals(String s) {
        if (this.name.toLowerCase() == s.toLowerCase()) return true;
        else return false;

    }
```

**Comment by Marame** [comment]
i used equals instead

**Comment by lfoppiano** [comment]
`LanguageCode.java:128` and around there are still `==`

**Comment by Marame** [comment]
fix#7
